internal/plugin/native/manager.go

```diff
@@ -486,6 +490,11 @@ func (rr *Manager) install(t plugin2.PluginType, name, src string, shellParas []
 			conf.Log.Infof(`run install script:%s`, outb.String())
 			conf.Log.Infof("install %s plugin %s", plugin2.PluginTypes[t], name)
 		}
+	} else if haveInstallParas && !haveInstallFile {
```

Comment:
Suggest to do this earlier to avoid copy then deleting files

Reply:
ack